tox/__init__.py

```diff
@@ -1,8 +1,10 @@
-#
-__version__ = '2.6.1.dev1'
+from pkg_resources import get_distribution
 
 from .hookspecs import hookspec, hookimpl  # noqa
 
+_version = get_distribution('tox').version
```

Comment:
oh, thanks @nicoddemus did not see that option yet. I will update the PR accordingly.